apps/store/themes/store/js/asset-core.js

```diff
@@ -4,8 +4,13 @@ var asset = {};
     var PROCESSING_TEXT = "processing";
     var SUCCESS_TEXT =  "success";
     var ERROR_TEXT = "error";
-    var getText = function(key) {
-        return $('#main-bookmark').data(key);
+    var getText = function (key) {
+        var bookmarkData = $('.main-bookmark');
+        var text = "";
```

Comment:
Thanks, I have change the default message accordingly [commit](https://github.com/wso2/carbon-store/pull/726/commits/ae5c84840d4d2b44979ea22a2305288388eb90a0)